tests/postgresql/needs.json

```diff
@@ -4,6 +4,7 @@
 			"repository": "git://git.postgresql.org/git/postgresql.git",
 			"commit": "0ed1e572b644e064e480e14d2f6afe03d11638a7",
 			"project": {
+				"max-jobs": 1,
```

Comment:
I thought the same thing until I realized that `-j` stands for exactly that: `--jobs`. I'm all for changing it, but I think it's worth considering the historical significance.